main.go

```diff
@@ -18,12 +23,6 @@ import (
 // @host localhost:1234
 // @BasePath /
 
-// User struct
-type User struct {
-	Name string `json:"name"`
-	UserID int64 `json:"userID"`
-}
-
 // Declare a local userCount
 var userCount int64
```

Comment:
Since the User struct was just an example and its getting removed, also remove these userCount so that we keep the code clean